src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
+  --Message-background: var(--color-gray);
+  --Message-offline-background: var(--color-orange);
+  --Message-offline-color: var(--color-white);
```

Comment:
> I just tested it and it fails for smaller text and passes for larger text.

Gotcha. When you say smaller text, do you mean when this pattern is shown at smaller sizes?

If it's just _barely_ below passing, then I'm okay proceeding. If we're failing pretty hardcore, though, then we're doing a disservice to folks with visual impairments.